tv_shows_app/src/styles/theme/components/inputs.ts

```diff
@@ -0,0 +1,33 @@
+import { defineStyleConfig } from '@chakra-ui/react';
+
+const Input = defineStyleConfig({
+  baseStyle: {
+    field: {
+      bg: 'purple2',
+      color: 'white',
+      borderRadius: '40px',
+      textStyle: 'button',
```

Comment:
Ovo mi je malo off.
Unutar teme za input koristim textStyle definirat pod nazivom button.
Rekli smo text style maknuti i razdvojiti to na fontSize, fontWeight i lineHeight foundation stilove